static/website/templates/get-question.html

```diff
@@ -149,10 +150,26 @@ <h4><u>Answers:</u></h4>
             </div>
         </div>
 
-        <div class="answer {% if user|is_moderator %}editable{% endif %}" id="answer{{answer.id}}" style="margin-left: 63px;">                               
-            
+        <div class="answer {% if user|is_moderator and MODERATOR_ACTIVATED or user|can_edit:answer %}editable{% endif %}" id="answer{{answer.id}}" style="margin-left: 63px;">                               
             <div class="body" id="body{{ answer.id }}" style="padding-bottom: 12px; padding-top: 15px">
-                <p>{{ answer.body| safe }}</p>
+                {% if user|is_moderator and MODERATOR_ACTIVATED or user|can_edit:answer and answer|havenot_comments %}
+                <form id="form{{ answer.id }}" method="POST"  action="{% url 'website:ajax_answer_update' %}">
+                    {% csrf_token %}
+                    <input type="hidden" name="answer_id" value="{{ answer.id }}">
+                <div class="edit-answer" id="ans{{ answer.id }}1">
+                    {% render_field form.body class+="form-control body" name="answer_body" id=answer.id %}
+                </div>
+                </form>
+                {% endif %}
+                <div id="ans{{ answer.id }}2">{{ answer.body| safe }}</div>
+                {% if user|is_moderator and MODERATOR_ACTIVATED or user|can_edit:answer and answer|havenot_comments %}
```

Comment:
use brackets here to make logic clear. use of **and** as well as **or** in the same conditional without brackets is not recommended.

Reply:
```suggestion
            {% if can_edit_answer(user, answer) %}
```

in that case, please define a function with appropriate parameters and call that function from here. Function as per current code (please fix and use):

```
def can_edit_answer(user, answer):
    if user|is_moderator and MODERATOR_ACTIVATED:
        return True
    if user|can_edit:answer and answer|havenot_comments:
        return True
    return False
```